VSCode/armsnippets.json

```diff
@@ -15,6 +15,30 @@
     ],
     "description": "Skeleton ARM Template"
 },
+"Azure Resource Manager (ARM) Parameters Template": {
```

Comment:
I'd rather not introduce a snippet for parameters. It should just be a single property ("parameters": {}), and the template snippet above already provides that.

Reply:
This snippet is intended for creating a new azuredeploy.parameters.json file, and adds the set of members recommended to have in that file. In difference from the one above, it also reference the correct schema (.../deploymentParameters.json vs. .../deploymentTemplate.json). Please correct me if I missunderstand your feedback

Reply:
Sorry my mistake. Good idea!